tests/integration/bond_test.py

```diff
@@ -648,3 +651,61 @@ def test_set_miimon_100_on_existing_bond(bond99_with_2_slaves):
     bond_config[Bond.OPTIONS_SUBTREE] = {"miimon": 100}
     libnmstate.apply(state)
     assertlib.assert_state_match(state)
+
+
+@pytest.fixture
+def eth1_eth2_with_no_profile():
+    iface_down(ETH1)
+    iface_down(ETH2)
+    yield
```

Comment:
```suggestion
    try:
        yield
    finally:
```

Reply:
Fixtures have is implicitly, therefore this is not needed.

Reply:
Thank you